arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -525,6 +528,28 @@
           "description": "An array of administrator user identities."
         }
       }
+    },
+    "BackupConfiguration": {
+      "description": "An object that represents backup configurations",
+      "type": "object",
+      "properties": {
+        "storageAccount": {
+          "type": "string",
+          "description": "The name of storage account for backup configuration. It is in the format of 'Resource Group Name/Storage Account Name'"
```

Comment:
@johanste Just synced with the team, we do have a plan to change the behavior to use full resource id for storage account. So I updated the description accordingly. Thanks.